modules/custom/activity_send/modules/activity_send_email/src/Plugin/QueueWorker/ActivityDigestWorker.php

```diff
@@ -127,7 +134,7 @@ public function processItem($data) {
           if (
           ($activity instanceof Activity) &&
             ($activity->getRelatedEntity() !== NULL) &&
-            !$activity->getRelatedEntity()->access('view', $target)
+          ($activity->getRelatedEntity()->access('view', $target) === FALSE)
```

Comment:
@navneet0693 could you fix plz the inconsistent indentation on these statements?